src/ctapipe/core/provenance.py

```diff
@@ -85,18 +217,18 @@ def start_activity(self, activity_name=sys.executable):
         activity.start()
         self._activities.append(activity)
         log.debug(f"started activity: {activity_name}")
+        return activity
```

Comment:
This new return is not mentioned in the class or method doc strings, so they need updating to let people know they can start an activity and immediately use it. 

Though it seems like there's not many places `start_activity` is used outside of the tests anyway.

Reply:
This is mostly an internal method to be called when a `Tool` starts up. It made my linter a bit happier to return here instead of calling this and then return `self.current_activity` in the method below.